scripts/game_structure/load_cat.py

```diff
@@ -182,6 +184,7 @@ def json_load():
                 )
 
             # new_cat.pronouns = cat["pronouns"] if "pronouns" in cat else [new_cat.default_pronouns[0].copy()]
+            # new_cat.pronouns = cat.get("pronouns", new_cat.default_pronouns[0].copy())
```

Comment:
If these lines are no longer necessary, and replaced by line 125, could you delete them to keep the loading code tidy? Same with line 124.

Reply:
Done :D